pages/learn/glossary.md

```diff
@@ -137,6 +141,14 @@ The following are considered tools:
 
 Applications which use JSON Schema internally without exposing that functionality in some way, for example, validating configuration files or web requests, are not considered tooling.
 
+### validation result
+
+The validation result in the context of JSON Schema refers to the outcome of applying the entire JSON Schema to the entire instance document. This application should yield a boolean assertion, indicating whether the instance document conforms to the rules and constraints specified in the schema. The validation result signifies whether the instance document passes or fails validation against the [schema](#schema).
+
+#### See also
+
+[Get started with JSON schema](../blog/posts/get-started-with-json-schema-in-node-js.md)
```

Comment:
I don't think this 'see also' belongs on the 'validation result' entry - it might be appropriate to mention in the intro, but probably most, if not all, of this glossary relates to getting started.